<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PPMM</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/icon100.png</normaloff>images/icon100.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="Buttons">
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="styleSheet">
         <string notr="true">image: url(:/images/Save22.png);</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/Save22.png</normaloff>:/images/Save22.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Load</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/Open22.png</normaloff>:/images/Open22.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="assembleButton">
        <property name="text">
         <string>Assemble</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/Assemble22.png</normaloff>:/images/Assemble22.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/Play22.png</normaloff>:/images/Play22.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stepButton">
        <property name="text">
         <string>Step</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/StepForward22.png</normaloff>:/images/StepForward22.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>164</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="editTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="executeTab">
       <attribute name="title">
        <string>Execute</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="codeBrowser"/>
        </item>
        <item>
         <widget class="QTableWidget" name="memTable">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>195</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>195</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Endere√ßo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Valor</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QTableWidget" name="registersTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>80</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>17</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>$zero</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>$at</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>$v0</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>$v1</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>$a0</string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>$a1</string>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>$a2</string>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string>$a3</string>
       </property>
      </item>
      <item row="7" column="1">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="8" column="0">
       <property name="text">
        <string>$t0</string>
       </property>
      </item>
      <item row="8" column="1">
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item row="8" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="9" column="0">
       <property name="text">
        <string>$t1</string>
       </property>
      </item>
      <item row="9" column="1">
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item row="9" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="10" column="0">
       <property name="text">
        <string>$t2</string>
       </property>
      </item>
      <item row="10" column="1">
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item row="10" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="11" column="0">
       <property name="text">
        <string>$t3</string>
       </property>
      </item>
      <item row="11" column="1">
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item row="11" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="12" column="0">
       <property name="text">
        <string>$t4</string>
       </property>
      </item>
      <item row="12" column="1">
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item row="12" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="13" column="0">
       <property name="text">
        <string>$t5</string>
       </property>
      </item>
      <item row="13" column="1">
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item row="13" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="14" column="0">
       <property name="text">
        <string>$t6</string>
       </property>
      </item>
      <item row="14" column="1">
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item row="14" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="15" column="0">
       <property name="text">
        <string>$t7</string>
       </property>
      </item>
      <item row="15" column="1">
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item row="15" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="16" column="0">
       <property name="text">
        <string>$s0</string>
       </property>
      </item>
      <item row="16" column="1">
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item row="16" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="17" column="0">
       <property name="text">
        <string>$s1</string>
       </property>
      </item>
      <item row="17" column="1">
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item row="17" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="18" column="0">
       <property name="text">
        <string>$s2</string>
       </property>
      </item>
      <item row="18" column="1">
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item row="18" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="19" column="0">
       <property name="text">
        <string>$s3</string>
       </property>
      </item>
      <item row="19" column="1">
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item row="19" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="20" column="0">
       <property name="text">
        <string>$s4</string>
       </property>
      </item>
      <item row="20" column="1">
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item row="20" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="21" column="0">
       <property name="text">
        <string>$s5</string>
       </property>
      </item>
      <item row="21" column="1">
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item row="21" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="22" column="0">
       <property name="text">
        <string>$s6</string>
       </property>
      </item>
      <item row="22" column="1">
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item row="22" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="23" column="0">
       <property name="text">
        <string>$s7</string>
       </property>
      </item>
      <item row="23" column="1">
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item row="23" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="24" column="0">
       <property name="text">
        <string>$t8</string>
       </property>
      </item>
      <item row="24" column="1">
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item row="24" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="25" column="0">
       <property name="text">
        <string>$t9</string>
       </property>
      </item>
      <item row="25" column="1">
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item row="25" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="26" column="0">
       <property name="text">
        <string>$k0</string>
       </property>
      </item>
      <item row="26" column="1">
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item row="26" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="27" column="0">
       <property name="text">
        <string>$k1</string>
       </property>
      </item>
      <item row="27" column="1">
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item row="27" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="28" column="0">
       <property name="text">
        <string>$gp</string>
       </property>
      </item>
      <item row="28" column="1">
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item row="28" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="29" column="0">
       <property name="text">
        <string>$sp</string>
       </property>
      </item>
      <item row="29" column="1">
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item row="29" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="30" column="0">
       <property name="text">
        <string>$fp</string>
       </property>
      </item>
      <item row="30" column="1">
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item row="30" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="31" column="0">
       <property name="text">
        <string>$ra</string>
       </property>
      </item>
      <item row="31" column="1">
       <property name="text">
        <string>31</string>
       </property>
      </item>
      <item row="31" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="OutputText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
